### PR TITLE
Add homePageUrl to registry-client and change status route to info

### PR DESCRIPTION
**Pull request by danielbayerlein**

### CODE REVIEW

File: honeycomb-registry-client/index.js [hunk 1]
```diff
@@ -26,7 +28,7 @@ function register(name, host, port) {
 
   const eureka = {
     host: process.env.EUREKA_HOST || 'localhost',
-    port: process.env.EUREKA_PORT || '1111',
+    port: process.env.EUREKA_PORT || '8761',
```
Comment (chrishelgert) [inline honeycomb-registry-client/index.js]: why port 8761?

File: honeycomb-registry-client/index.js [hunk 2]
```diff
@@ -17,7 +17,9 @@ function register(name, host, port) {
     app: name,
     hostName: host,
     vipAddress: process.env.EUREKA_VIP || host,
-    statusPageUrl: `http://${host}:${port}`,
+    statusPageUrl: `http://${host}:${port}/info`,
```
Comment (chrishelgert) [inline honeycomb-registry-client/index.js]: 👍

File: honeycomb-registry-client/index.js (hunk 1 above)
Comment (danielbayerlein) [inline honeycomb-registry-client/index.js]: The default port in each tutorial, see https://github.com/spring-cloud-samples/eureka

Comment (chrishelgert) [inline honeycomb-registry-client/index.js]: Then looks good 😎

**Pull request merged**